virt/install/preparing-cluster-for-virt.adoc

```diff
@@ -70,6 +80,60 @@ include::modules/virt-aws-bm.adoc[leveloffset=+2]
 * xref:../../virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc#virt-connecting-vm-to-ovn-secondary-network[Connecting a virtual machine to an OVN-Kubernetes secondary network]
 * xref:../../virt/vm_networking/virt-exposing-vm-with-service.adoc#virt-exposing-vm-with-service[Exposing a virtual machine by using a service]
 
+// Hiding in ROSA/OSD - todo: double check this
+ifndef::openshift-rosa,openshift-dedicated[]
+[id="ibm-z-linuxone-compatibility_{context}"]
+=== {ibm-z-title} and {ibm-linuxone-title} compatibility
+
+You can use {VirtProductName} in an {product-title} cluster that is installed in a logical partition (LPAR) on an {ibm-z-name} or {ibm-linuxone-name} (s390x architecture) system.
+
+ifdef::openshift-enterprise[]
+:FeatureName: Using {VirtProductName} in a cluster deployed on s390x architecture
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
+
+Some features are not currently available on s390x architecture, while others require workarounds or procedural changes. These lists are subject to change.
+
+[discrete]
+[id="currently-unavailable-ibm-z_{context}"]
+==== Currently unavailable features
+
+The following features are not available or do not function on s390x architecture:
+
+* Memory hot plugging and hot unplugging
+* Watchdog devices
+* Node Health Check Operator
+* SR-IOV Operator
+* virtual Trusted Platform Module (vTPM) devices
+* {pipelines-title} tasks
+* UEFI mode for VMs
+* PCI passthrough
+* USB host passthrough
+* Configuring virtual GPUs
+* {VirtProductName} cluster checkup framework
```

Comment:
Discussed with @nbziouec that with IBM Z creating and managing VMs (listed [here](https://88929--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/about_virt/about-virt#virt-what-you-can-do-with-virt_about-virt)) only works with Linux
```suggestion
* {VirtProductName} cluster checkup framework
* Create and manage Windows VMs
```